app/models/edition.rb

```diff
@@ -227,32 +227,6 @@ def web_url
     Plek.current.website_root + base_path
   end
 
-  # FIXME These are required for the special 'description' column but should
-  # be removed when the column is fixed.
-  def description=(value)
```

Comment:
If we want to allow full rollback ability we'd probably want to keep this method and have it up date description_json, which would probably for the best as it would allow us to back out of this without lost data (hopefully anyway)